arkane/commonTest.py

```diff
@@ -270,24 +270,6 @@ def testTransitionStateStatmech(self):
         job.load()
 
 
-class TestStatmech(unittest.TestCase):
```

Comment:
Can you move this change to the next commit where you add statmechTest.py? The rest of this commit can probably be combined with one of your earlier syntax commits as well.